src/test/resources/features/Glossary.feature

```diff
@@ -1,11 +1,15 @@
 @ITSC
+@Glossary
 @Beta
 Feature: Glossary
 
   Background:
     Given On the feature switch page I check features
-      | Beta Features |
-    And I navigate to the Glossary page
+      | Beta Features        |
+      | Income and Deduction |
+    And I navigate to the Software Choices home page
+    And I click on the test software vendor name one link
+    And I click on the glossary link
```

Comment:
Shouldn't this finish with "Then I am on the glossary page"?

Reply:
It's fine, it's part of the background, the next thing which would be run in the scenario is "On the glossary page..." which does that check of being on the glossary page